src/aerie_cli/aerie_client.py

```diff
@@ -1386,6 +1427,21 @@ def upload_scheduling_goal(self, model_id, name, definition):
 
         return resp["id"]
 
+    def upload_scheduling_goals(self, upload_object):
+        upload_scheduling_goals_query = """
+        mutation InsertGoals($input:[scheduling_goal_insert_input!]!){
+            insert_scheduling_goal(objects: $input){
+                returning {id}
+            }
+        }"""
+        
+        resp = self.host_session.post_to_graphql(
+            upload_scheduling_goals_query,
+            input=upload_object
+        )
+
+        return resp
```

Comment:
The other upload methods don't return the raw response JSON but rather an object containing that data (so for this method, an array of ids). Could you please update your methods to reflect that?